frontend/src/service/useApi.ts

```diff
@@ -0,0 +1,47 @@
+import {axiosAuthenticated} from "./axios";
+
+export type BotInfo = {
+    nyxxVersion: string;
+    version: string;
+    platform: string;
+    memoryUsageString: string;
+    cachedChannels: number;
+    cachedMessages: number;
+    cachedGuilds: number;
+    cachedUsers: number;
+    cachedVoiceStates: number;
+    shardCount: number;
+    totalTagsCount: number;
+    totalReminderCount: number;
+    uptime: string;
+    docsUpdate: string;
+};
+
+export type Guild = {
+    id: string,
+    name: string,
+    banner?: string,
+    icon?: string,
+    cachedMembers: number,
+    cachedChannels: number,
+    cachedMessages: number,
+    cachedRoles: number,
+    enabledFeatures: string[],
+    tagsCount: number,
+};
+
+export class Api {
```

Comment:
What's the point of this class?
This can easily be replaced by a plain object with methods instead,